examples/app.py

```diff
@@ -24,6 +24,20 @@
 
 r"""Minimal Flask application example for development.
 
+Install the Invenio default theme
+
+You should execute these commands in the examples-directory.
+
+.. code-block:: console
+   $ pip install invenio-theme
```

Comment:
please move it to `examples/requirements.txt` (see https://github.com/inveniosoftware/invenio-marc21/tree/master/examples)